torch/_dynamo/variables/constant.py

```diff
@@ -104,7 +104,8 @@ def getitem_const(self, arg: VariableTracker):
     def is_literal(obj):
         if type(obj) in (int, float, bool, type(None), str, Ellipsis.__class__):
             return True
-        if type(obj) in (list, tuple, set, frozenset):
+        # The structure within is_literal get routed to variables.BaseListVariable
+        if type(obj) in (list, tuple, set, frozenset, torch.Size):
```

Comment:
note that unlike the other container types, entries in torch.Size can *only* be ints (other entries are rejected). This has caused not considerable grief when dealing with SymInts/pytree'ing. I don't know if it is relevant here; it is hard to tell what the downstream effects are.

@jansel, ngl, I still have no idea what a "literal" is, in your definition haha.

Reply:
This should be safe, protected during constant construction